DateFormat.ts

```diff
@@ -0,0 +1,31 @@
+import * as isoly from "isoly"
+export type DateFormat = "YYYY-mm-dd" | "mm/dd/YYYY" | "dd/mm/YYYY" | "dd.mm.YYYY"
+
+export namespace DateFormat {
+	export function fromLocale(locale: DateFormat | isoly.Locale | undefined): DateFormat {
+		let result: DateFormat
+		switch (locale) {
+			default:
```

Comment:
I generally put the default statement last. This will still work, but may be less consistent with other switch statements in the code.